### DISCUSSION TITLE
Requesting every long readme file owners

### DISCUSSION THREAD

**Discussion by Biniben** [community/community#43095] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

please have back to top icon cos it's too long to back to top
it would be very nice if you could refer to discussion #42712

**Comment by jfullstackdev** [comment, marked as answer]
you can use the Topics Summary by clicking the README.md icon, like this 

![image](https://user-images.githubusercontent.com/47092464/210247356-015d48f9-2b5b-4b8e-b35b-f6eb2b8df6af.png)
